### PR TITLE
Sync: Deprecate Queue Full Sync 💥💥💥💥💥💥

### PR DESCRIPTION
**Pull request by lezama**
This will probably need a hundred rebases before we can merge it but we can still use it as a reminder of things to delete. Enjoy!

### CODE REVIEW

**Comment by Rae28** [comment]
Test

**Comment by zinigor** [comment]
I'm moving this to the next milestone, thanks for discussing it with me, @tyxla and @mdbitz !

**Comment by kraftbj** [comment]
Rebased it against master.

File: class.jetpack-cli.php [hunk 1]
```diff
@@ -883,7 +882,7 @@ public function sync( $args, $assoc_args ) {
 							WP_CLI::log( sprintf( __( 'Reset Regular Sync Queue on %s', 'jetpack' ), get_site_url() ) );
 							break;
 						case 'full':
-							$listener->get_full_sync_queue()->reset();
+							// TODO reset full sync progress?.
```
Comment (kraftbj) [inline class.jetpack-cli.php]: It would be nice to have this somewhere. If not in the CLI, in the debug data?

**Comment by mdbitz** [comment]
Updated to 8.3 milestone. Immediate Full Sync is rolling out in 8.2 and we need to verify no regressions before removing the existing Sync logic.

**Comment by lezama** [comment]
🙇

**Pull request closed without merging**